example/cifar10/code/test.py

```diff
@@ -0,0 +1,45 @@
+from model import Net
+import torch
+import torchvision.transforms as transforms
+from PIL import Image
+import numpy as np
+from data_slicer import unpickle
+
+DATA_PATH = '../data/cifar-10-batches-py/test_batch'
```

Comment:
use abstract path? otherwise, the code will depend on the cmd path.
some code example:
```python
import os
_ROOT_DIR = os.path.dirname(os.path.dirname(__file__))
_DATA_PATH = os.path.join(_ROOT_DIR, "data/cifar-10-batches-py/test_batch")
```

Reply:
sure , thanks